src/poliastro/tests/test_czml.py

```diff
@@ -1,21 +1,32 @@
-import pytest  # noqa: E402 isort:skip
+"""
+This is a set of unit-tests for CZMLExtractor, which uses czml3 package to export
+poliastro data into CZML format used by Cesium. To run those tests, you need to have
+czml3 package installed.
+"""
+
 import sys
 
 from astropy import units as u  # noqa: E402
 
 from poliastro.bodies import Mars
 from poliastro.examples import iss, molniya  # noqa: E402
 
+import pytest  # noqa: E402 isort:skip
```

Comment:
Different linting tools make different decisions. We are already using Flake8 (which in turn uses PyFlakes), so let's not introduce another conflicting tool (and leave whatever isort thinks it's appropriate)

Reply:
Sure. Rebased and removed unnecessary commits.

Reply:
Notice that there's another `import pytest` in line 1, this is making the quality check fail

Reply:
`src/poliastro/tests/test_czml.py:9:1: F811 redefinition of unused 'pytest' from line 1`